src/data/astarte.rs

```diff
@@ -18,19 +18,29 @@
  * SPDX-License-Identifier: Apache-2.0
```

Comment:
This file is the counterpart of `astarte_message_hub_node.rs` that uses the SDK right?
It might be clearer to rename it to something that indicates that. Like `astarte_sdk_device`.